core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_responses.html

```diff
@@ -99,7 +99,7 @@
       </button>
     </div>
   </md-card>
-  <md-card style="margin: 20px 0px; padding: 0px;" ng-if="!isCurrentInteractionTrainable()">
+  <md-card style="margin: 20px 0px; padding: 0px;" ng-if="!isCurrentInteractionTrainable()"></md-card>
```

Comment:
Ah, yes. I missed it. But if this closing tag is removed then `+ Add New Oppia Response` becomes invisible if `SHOW_TRAINABLE_UNRESOLVED_ANSWERS` is set to True. Normally this should not happen, right?
I think the `ng-if` attribute should be removed from this tag. Because whether current interaction is trainable or not `+ Add New Oppia Response` should always be visible, right?